modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/checkpoint/CheckpointDirtyPages.java

```diff
@@ -108,33 +100,54 @@ public IgniteConcurrentMultiPairQueue<PersistentPageMemory, GroupPartitionId> to
      * @param partId Partition ID.
      */
     public @Nullable CheckpointDirtyPagesView findView(int grpId, int partId) {
```

Comment:
I see that you have not renamed this method. "View" is a very general term

Reply:
Why do we need two versions of the method now? Don't we always have a page memory instance when we need a "view"?

Reply:
Okay, what name should I use? I don’t know yet whether both methods will be needed or only one, do you insist on deleting the method for the general case?

Reply:
If you don't like the name, suggest a suitable one:
- getPartitionView - it's about partitions and not dirty page ids, does it really fit?

Reply:
Fix it.